static/app/views/organizationGroupDetails/allEventsTable.tsx

```diff
@@ -88,6 +88,7 @@ const AllEventsTable = (props: Props) => {
       }}
       transactionName=""
       columnTitles={columnTitles.slice()}
+      referrer="api.issues.issue_events"
```

Comment:
Without giving too much thought, Would be sweet if these are autogenerated, perhaps based on the url.

Reply:
yeah, it's not always the url though. sometimes we use referrers from within the backend too

Reply:
but yeah there's definitely room for improvement